woocommerce/woocommerce-admin
# Change event name to reflect plugin name

Fixes https://github.com/woocommerce/google-listings-and-ads/issues/840

To avoid reporting issues the plugin name should be used when installing the extension. This PR changes the event name in the onboarding flow from `install_google_ads` to `install_google_listings_and_ads`

### Screenshots
![image](https://user-images.githubusercontent.com/11388669/123962431-5e510300-d9a9-11eb-95e1-b5de4901c72f.png)

![image](https://user-images.githubusercontent.com/11388669/123962033-ea165f80-d9a8-11eb-976d-ab0f25a682a3.png)

### Detailed test instructions:
- Open the DevTool console and run `localStorage.setItem( 'debug', 'wc-admin:*' );` to enable tracking debugging mode (make sure all levels are being logged including verbose)
- Confirm we do not have the plugin Google Listing and Ads installed
- Go through the OnBoarding Wizard and complete each step
- At the include free features step, uncheck all extensions except Google Listings and Ads
- Click continue and confirm that the event `wcadmin_storeprofiler_store_business_features_continue` is logged in the console, and the extension name is `install_google_listings_and_ads`

Note: I also had to run the following snippet in the console to be able to expand the output before a page redirect.
```js
window.onbeforeunload = function (e) {
    console.log(e);
    return true;
}
```
